sass/main.scss

```diff
@@ -343,6 +343,59 @@ nav {
         }
 
     }
+    #modal {
+        display: none;
+        position: fixed;
+        z-index: 100;
+        padding-top: 27px;
+        left: 0;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        overflow: auto;
+        background-color:rgba(0,0,0) ;
+        background-color: rgba(0,0,0,0.9);
+    }
+
+    #modal-content {
+        margin: auto;
+        display: block;
+        width: 80%;
+        max-width: 700px;
+        -webkit-animation-name: zoom;
+        -webkit-animation-duration: 0.6s;
+        animation-name: zoom;
+        animation-duration: 0.6s;
+    }
+
+    @-webkit-keyframes zoom {
```

Comment:
Parcel comes with [PostCSS](https://postcss.org/) which adds prefixes for us automatically. You can remove any and all `-webkit-` (and other) browser prefixes 😄